### ISSUE TITLE
Markdown

### ISSUE THREAD

**Issue by Tiemichael** [turtl/js#186]
I understand from previous issues that Turtl has implemented markdown feature explained [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Here-Cheatsheet) .

Besides Line Breaks (and Syntax Highlighting) everything seems to work fine. 
Hitting 'ENTER' once does not provide a new line, hitting 'ENTER' twice seems to start a new paragraph.

Not sure If I misunderstand something, or this Markdown feature is not implemented ...
(I am using Turtl on my own server (running on a Raspberry PI), and using Windows, android and a Web-Client.)

**Comment by ldidry** [comment]
> Hitting 'ENTER' once does not provide a new line, hitting 'ENTER' twice seems to start a new paragraph.

Yes, it's usual markdown. If you want a new line, keep two spaces at the end of the line and hit Enter.
